.github/LICENSE.TXT

```diff
@@ -0,0 +1 @@
+gdal/LICENSE.TXT
```

Comment:
this should be `../gdal/LICENSE.TXT` surely?

Reply:
As fare as I understood from [this](https://mokacoding.com/blog/symliks-in-git/) it suppose to be relative to the root of the repo.

Reply:
I literally checked out the branch and ran `ls` in it above → the symlink is broken.

```
$ head .github/LICENSE.TXT
head: cannot open '.github/LICENSE.TXT' for reading: No such file or directory
```

Does it work for you?

Reply:
Thanks, I'll recheck it and report back soon.